presto-docs/src/main/sphinx/release/release-0.199.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.199
+=============
+
+General Changes
+---------------
+
+* Allow users to create views for their own use when they do not have permission
+  to grant others access to the underlying tables or views. To enable this,
+  creation permission is now only checked at query time, not at creation time,
+  and the query time check is skipped if the user is the owner of the view.
+* Add support for spatial left join.
+* Add :func:`hmac_md5`, :func:`hmac_sha1`, :func:`hmac_sha256`, and :func:`hmac_sha512` functions.
+* Add :func:`array_sort` function that takes a lambda as a comparator.
+* Add :func:`line_locate_point` geospatial function.
+* Add support for ``ORDER BY`` clause in aggregations for queries that use grouping sets.
+* Add support for yielding when unspilling an aggregation.
+* Expand grouped execution support to ``GROUP BY`` and ``UNION ALL``, making it possible
+  to execute aggregations with less peak memory usage.
+* Change the signature of ``round(x, d)`` and ``truncate(x, d)`` functions so that
+  ``d`` is of type ``INTEGER``. Previously, ``d`` could be of type ``BIGINT``.
+  This behavior can be reverted with the ``deprecated.legacy-round-n-bigint`` config option
+  or the ``legacy_round_n_bigint`` session property.
+* Optimize the :func:`ST_Intersection` function for rectangles aligned with coordinate axes
+  (e.g., polygons produced by the :func:`ST_Envelope` and :func:`bing_tile_polygon` functions).
+* Finish joins early when possible if one side has no rows. This happens for
+  either side of an inner join, for the left side of a left join, and for the
+  right side of a right join.
+* Improve predicate evaluation performance during predicate pushdown in planning.
+* Improve the performance of queries that use ``LIKE`` predicates on the columns of ``information_schema`` tables.
+* Improve the performance of map-to-map cast.
+* Improve the performance of :func:`ST_Touches`, :func:`ST_Within`, :func:`ST_Overlaps`, :func:`ST_Disjoint`,
+  and :func:`ST_Crosses` functions.
+* Improve the serialization performance of geometry values.
+
+Server RPM Changes
+------------------
+
+* Add support for installing Presto server RPM to machines with OpenJDK.
```

Comment:
I think we can simplify this to (as the "Presto server RPM" part is implied based on the section title)
```
Add support for installing on machines with OpenJDK.